src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  border: #f3f3f3 1px solid;
+  border-radius: 5px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.text {
+  height: 36px;
+  color: #060b35;
+  margin: 0 16px;
+  line-height: 18px;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 0 0 16px;
+  line-height: 14px;
+  width: 98px;
+}
+
+.stars {
+  display: flex;
+  width: 166px;
+  margin: 16px 0 0 16px;
+}
+
+.stars__stars {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+  background-image: url(/src/images/star.svg);
```

Comment:
The `background-image` property uses an absolute path starting with `/src/images/star.svg`. This could cause issues if the file structure changes or if the project is deployed to a different environment. Consider using relative paths or a variable to manage paths more flexibly.